### PR TITLE
[website] fix: pulsar website cli docs build error

### LINKED ISSUE #13452: [Website] `pulsar` CLI website shows commands of `pulsar-perf`
[`pulsar` 2.10 page](https://pulsar.apache.org/tools/pulsar/2.10.0-SNAPSHOT/) shows the commands of `pulsar-perf`, this is incorrect.

image

### PR DESCRIPTION
**Pull request by urfreespace**
Signed-off-by: LiLi <urfreespace@gmail.com>

*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #13452 

image

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Comment by urfreespace** [comment]
@Anonymitaet @tuteng PTAL, thanks.

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java [hunk 1]
```diff
@@ -89,10 +88,10 @@
     CmdGenerateDocumentation generateDocumentation;
 
     public PulsarClientTool(Properties properties) {
-        this.serviceURL = StringUtils.isNotBlank(properties.getProperty("brokerServiceUrl"))
+        this.serviceURL = isNotBlank(properties.getProperty("brokerServiceUrl"))
```
Comment (tuteng) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java:91]: It seems that these two packages `isNotBlank` and `isBlank` should be imported

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java [hunk 2]
```diff
@@ -142,7 +141,7 @@ private void updateConfig() throws UnsupportedAuthenticationException {
                 .tlsTrustStorePath(tlsTrustStorePath)
                 .tlsTrustStorePassword(tlsTrustStorePassword);
 
-        if (StringUtils.isNotBlank(proxyServiceURL)) {
+        if (isNotBlank(proxyServiceURL)) {
```
Comment (tuteng) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java:144]: It seems that these two packages `isNotBlank` and `isBlank` should be imported

**Review by tuteng — APPROVED** [review]

**Pull request merged**